samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/AuthorActionListener.java

```diff
@@ -185,7 +185,23 @@ public void processAction(ActionEvent ae) throws AbortProcessingException
 		author.setEditPubAssessmentRestricted(true);
 	}
     } 
+
+	String s2 = ServerConfigurationService.getString("samigo.editPubAssessment.restricted.afterStart");
```

Comment:
I think these would have been clearer if they used getBoolean and had a specific defined default value.